tests/test_subset_graph.py

```diff
@@ -230,6 +230,43 @@ def test_save_load_merge(self):
         n_partitions = ht.output_partitions(filename, outfile)
         assert n_partitions == 1, n_partitions        # combined.
 
+    def test_save_load_merge_truncate(self):
+        ht = khmer.new_hashbits(20, 4 ** 4 + 1)
+        filename = utils.get_test_data('test-graph2.fa')
+
+        (total_reads, total_kmers) = ht.consume_fasta_and_tag(filename)
+        assert total_reads == 3, total_reads
+
+        divvy = ht.divide_tags_into_subsets(1)
+        print divvy
+        assert len(divvy) is 3
+        (a, b, c) = divvy
+
+        outfile1 = utils.get_temp_filename('x.pmap')
+        outfile2 = utils.get_temp_filename('y.pmap')
+
+        x = ht.do_subset_partition(a, b)
+        ht.save_subset_partitionmap(x, outfile1)
+        del x
```

Comment:
Not really necessary in python; `del` doesn' actually free the memory, just notifies the gc that it *can* be freed